whylog/assistant/__init__.py

```diff
@@ -0,0 +1,91 @@
+import abc
+
+
+class AbstractAssistant(object):
```

Comment:
`ABCMeta` metaclass setting is missing, so `abstractmethod` doesn't really do anything.